### DISCUSSION TITLE
Positive about Slingplayer and Slingcatcher, that still working

### DISCUSSION THREAD

**Discussion by Alimond52** [GerryDazoo/Slinger#612] [General]
Strangely, Slingplayer 2.0.4.522 and Slingcatcher continued to work as well, even over the Internet. I configured fixed IP address, from the Slingbox. Then started Slingplayer for Windows, either with or without being an administrator. After a few tries the power starts and it works as usual as it did in the past, same thing with the Slingcatcher. For Internet streaming one must have logged in to Sling account first.
The only thing I'm missing is the exact virtual remote control for my STB box that was made in 2012. It disappeared when my PRO-HDs got bricked. Setup Assistant cannot be accessed as Flashplayer is no longer available. Or it complains that there is no Internet connection. Which is wrong.
Have any of you managed to solve this?
In the older Slingplayern version 1.5.1.343 which is used after Brickning, my Remote Control is unfortunately not included.
I have the .spr file.
Thanks for all the tips
I wish you a nice summer

**Comment by GerryDazoo** [comment]
This is interesting. I've had another report recently that the Sling player software was working for place-shifting.
I think you'll need to find a compatible custom remote definition file for SlingPlayer to use.

**Comment by Alimond52** [comment]
Hi Gerry,
Yes Slingplayer works for Place Shifting. I didn't think it was possible anymore. But I tried for a very long time back and forth, and finally it worked. I think that it is important to be able to log in to the Sling **account** and then make sure to register the SB box with a **fixed** IP address.
As for the nice remote control created in 2012, for my STB Humax. The model designation of the remote control itself is "**RM-301**". Before November 2022, it was installed under **Satellite** when installing a remote control. But from what I can see, it can't be done anymore.
So what I did was to install a remote control from Slingplayer version 1.5.1.343, and selected "**Custom**" and installed the IR codes themselves from a file: "**V2010_PL.bin**"
And then you get a **Basic** remote where many of the various commands are only found in a long **list**, which is very impractical. I have the actual skin file for my Remote "RM-301", but it doesn't seem to be installed, it's called "**Humax RM-301.sp**r". So the look will be just a Basic remote, unfortunately. I don't know how to install this .spr file, so it works. Have tried most things.

**Reply by anonyada** [reply]
Perhaps you all are being lured in for another ECM…..

Just Say’n

**Reply by Alimond52** [reply]
The Sling account server has probably been shut down in **January** 2024.
It has worked perfectly to be able to see everything as usual even after November 2022, via Slingplayer, as I wrote above.
But in January 2024 it stopped working.
It is no longer possible to log in to the Sling account.
And then all Slingboxes are no longer listed in the Slingbox **Directory** except for one Slingbox in the **local** network. So it is not possible to see anything outside the local network.
So things went well in 2023, but now it is closed?
Are there more than me who discovered the same thing?
I'm guessing that SlingMedia has **shut** down those servers completely now and therefore we can't log in anymore?
